README.md

```diff
@@ -244,12 +246,12 @@ The valid theme configurations are:
 when configured with the `theme-file-name` which is a `yaml` file,
 `lsd` will look up the theme file in the following way:
 
-- relative name: check the themes under XDG Base Directory, e.g. ~/.config/lsd/themes/<theme-file-name>.yaml
```

Comment:
Colors themes are still under `themes`, just the `icons.yml` is in the base dir.